db/schema.rb

```diff
@@ -348,15 +348,14 @@
     t.integer "age2"
     t.integer "age2_known"
     t.string "relat2"
-    t.string "otherrelat2"
```

Comment:
Not sure why this was added back in as part of [this PR](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/commit/c52a30ae0c8cf335965340a1c03915d8068ee2a6), it shouldn't have been.